### ISSUE TITLE
flutter version 3.16.9 

### ISSUE THREAD

**Issue by gborri** [mingchen/docker-android-build-box#172]
hello, doy think is it possible to have a tag with 3.16.9 flutter version?
we have not yet tested software with 3.19 version.

let me know
thanks in advance
Giovanni

**Comment by master-bob** [comment]
I'm not sure if it is possible. In the meantime a solution is to build the image yourself with the specified flutter version.

**Comment by mingchen** [comment]
Thanks @master-bob
Created tag `1.27.0`.

**Comment by master-bob** [comment]
Something is not correct.

```console
$ docker run -it mingc/android-build-box:1.27.0 flutter --version
Flutter 3.19.1 • channel stable • https://github.com/flutter/flutter.git
Framework • revision abb292a07e (13 days ago) • 2024-02-20 14:35:05 -0800
Engine • revision 04817c99c9
Tools • Dart 3.3.0 • DevTools 2.31.1```

**Comment by master-bob** [comment]
There is a bug in the CI build system, #178 , so until that is fixed the tagged version 1.27.0 will not being using the 3.16.9 Flutter version.
